### ISSUE TITLE
Undefined index on ListenerFetcher

### ISSUE THREAD

**Issue by QuingKhaos** [egulias/ListenersDebugCommandBundle#22]
When executing the command, I get the following error:

```
PHP Notice:  Undefined index: method in <symfony>/vendor/egulias/listeners-debug-command-bundle/Listener/ListenerFetcher.php on line 81
PHP Stack trace:
PHP   1. {main}() <symfony>/app/console:0
PHP   2. Symfony\Component\Console\Application->run() <symfony>/app/console:26
PHP   3. Symfony\Bundle\FrameworkBundle\Console\Application->doRun() <symfony>/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:121
PHP   4. Symfony\Component\Console\Application->doRun() <symfony>/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:96
PHP   5. Symfony\Component\Console\Application->doRunCommand() <symfony>/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:191
PHP   6. Symfony\Component\Console\Command\Command->run() <symfony>/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:892
PHP   7. Egulias\ListenersDebugCommandBundle\Command\ListenersCommand->execute() <symfony>/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:241
PHP   8. Egulias\ListenersDebugCommandBundle\Command\ListenersCommand->outputListeners() <symfony>/vendor/egulias/listeners-debug-command-bundle/Command/ListenersCommand.php:84
PHP   9. Egulias\ListenersDebugCommandBundle\Listener\ListenerFetcher->fetchListeners() <symfony>/vendor/egulias/listeners-debug-command-bundle/Command/ListenersCommand.php:99
```

This error occours six times, which is exactly number of listeners without method (weird). These are the following ones:

| Name | Event | Method | Priority | Type | Class Name |
| --- | --- | --- | --- | --- | --- |
| `cmf_routing.phpcrodm_route_idprefix_listener` | `postLoad` |  | 0 | `listener` | `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\IdPrefixListener` |
| `cmf_routing.phpcrodm_route_idprefix_listener` | `postPersist` |  | 0 | `listener` | `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\IdPrefixListener` |
| `cmf_routing.phpcrodm_route_idprefix_listener` | `postMove` |  | 0 | `listener` | `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\IdPrefixListener` |
| `doctrine_phpcr.jackalope_doctrine_dbal.schema_listener` | `postGenerateSchema` |  | 0 | `listener` | `Doctrine\Bundle\PHPCRBundle\EventListener\JackalopeDoctrineDbalSchemaListener` |
| `sylius.listener.order_item_inventory` | `prePersist` |  | 0 | `listener` | `Sylius\Bundle\CoreBundle\EventListener\OrderItemInventoryListener` |
| `sylius.listener.order_item_inventory` | `onFlush` |  | 0 | `listener` | `Sylius\Bundle\CoreBundle\EventListener\OrderItemInventoryListener` |

As the event name supposes, these are special Doctrine lifecycle listeners, so they are not specific to Symfony CMF or Sylius.

**Comment by egulias** [comment]
Hi @patkar 
You are right. The definition (https://github.com/Sylius/SyliusCoreBundle/blob/master/Resources/config/services.xml#L270) lacks the standard for **symfony** event listeners, because they are doctrine ones.
I'll fix it ASAP, for v2, and 1.9.
Thanks for reporting!

**Comment by egulias** [comment]
See https://github.com/egulias/ListenersDebug/commit/e1e76541b9e71b2207385ee9e8f84357adde1c70
As is the lib the one that supports now the fetching.
@patkar can you upgrade to the v2.0 of the bundle and do a composer update, the pach should work.
Let me know if everything went fine.

**Comment by egulias** [comment]
@patkar ping

**Comment by egulias** [comment]
This has been fixed for 2.1, please upgrade!
